gamestonk_terminal/stocks/discovery/disc_controller.py

```diff
@@ -352,6 +356,90 @@ def call_under(self, other_args: List[str]):
         except Exception as e:
             print(e, "\n")
 
+    def call_gtech(self, other_args: List[str]):
+        """Process gtech command"""
+        parser = argparse.ArgumentParser(
+            add_help=False,
+            formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+            prog="gtech",
+            description="Print up to 25 top ticker losers. [Source: Yahoo Finance]",
+        )
+        parser.add_argument(
+            "-n",
+            "--num",
+            action="store",
+            dest="num",
+            type=check_int_range(1, 25),
+            default=5,
+            help="Number of the undervalued stocks to retrieve.",
+        )
+        parser.add_argument(
+            "--export",
+            choices=["csv", "json", "xlsx"],
+            default="",
+            type=str,
+            dest="export",
+            help="Export dataframe data to csv,json,xlsx file",
+        )
+        try:
+            if other_args:
+                if "-" not in other_args[0]:
+                    other_args.insert(0, "-n")
+
+            ns_parser = parse_known_args_and_warn(parser, other_args)
+            if not ns_parser:
+                return
+
+            yahoofinance_view.display_gtech(
```

Comment:
Perfect.

Unrelated, I wrote as a convention on the contributing document that we could have:
`plot_` for a plot
`display_` for a table
`print_` for a string

What do you think?

Reply:
I think it's a great idea!